Makefile

```diff
@@ -40,6 +40,7 @@ deploy: $(TOOLBIN)/kubectl $(TOOLBIN)/helm
 	$(TOOLBIN)/helm install fybrik charts/fybrik --values $(VALUES_FILE) \
                --namespace $(KUBE_NAMESPACE) --wait --timeout 120s
 
+.PHONY: pre-test
```

Comment:
According to https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html, 
`a phony target should not be a prerequisite of a real target file`.
`pre-test` is a prerequisite of `test`

Reply:
@roytman ^^^